bee-network/bee-gossip/Cargo.toml

```diff
@@ -1,15 +1,13 @@
 [package]
-name = "bee-network"
-version = "0.2.2"
+name = "bee-gossip"
+version = "0.3.0"
```

Comment:
This is technically a new crate (new name), shouldn't it be 0.1.0 ?

Reply:
good question: isn't it technically the same crate, but only theoretically a new one due to the rebranding? :thinking:
Anyway....I don't know what's correct here. I would find it odd, though, if they a crate with a long history at say version 8.1.3 would suddenly become 0.1.0 again due to a rebranding. Is Meta(Facebook) 0.1.0 again? :trollface: 

 Your choice :slightly_smiling_face: